### DISCUSSION TITLE
unsung/

### DISCUSSION THREAD

**Discussion by giscus** [zmike/zmike.github.io#18] [Announcements]
# unsung/

Long Time No See

https://www.supergoodcode.com/unsung/

**Comment by rhysperry111** [comment]
Omg he wrote a whole section about me..... 😍

**Comment by qwertychouskie** [comment]
Good to finally get another post, I had honestly stopped checking for new posts after a while.  It's been almost a month since the previous post, and that post had a whopping word count of 5.  (7 if you count the titles).

**Reply by zmike** [reply]
too hot to post

**Reply by qwertychouskie** [reply]
https://www.homedepot.com/p/Whynter-12-000-BTU-Dual-Hose-Portable-Air-Conditioner-with-Dehumidifier-and-3M-SilverShield-Filter-in-Black-ARC-126MDB/316011024

Can confirm, very good

**Comment by jams3223** [comment]
The man is back.
